File: src/com/kostu96/bots/entities/animals/Sheep.java
package com.kostu96.bots.entities.animals;

import com.kostu96.bots.World;
import com.kostu96.bots.entities.Entity;
import com.kostu96.bots.utils.Point;

public class Sheep extends Animal {
	private static final int STRENGTH = 4;
	private static final int INITIATIVE = 4;

	public Sheep(World world, Point position) {
		super(world, position, STRENGTH, INITIATIVE);
	}

	@Override
	public void collision(Entity other) {

	}

}

File: src/entities/sheep.py
from .animal import Animal

class Sheep(Animal):
    color = (250, 250, 250)
    symbol = 'SH'

    def __init__(self, world, position):
        super().__init__(world, position, 4, 4)

    def __str__(self):
        return 'Sheep'

    def getColor(self):
        return Sheep.color

    def getSymbol(self):
        return Sheep.symbol
